cli/src/main.rs

```diff
@@ -143,7 +143,7 @@ enum Cli {
     #[structopt(
         name = "keygen",
         alias = "k",
-        about = "Creates a new agent key pair, asks for a passphrase and writes an encrypted key bundle to ~/.config/holochain/keys"
+        about = "Creates a new agent key pair, asks for a passphrase and writes an encrypted key bundle to disk in a location specific to Holochain (and dependent on OS platform)"
```

Comment:
```suggestion
        about = "Creates a new agent key pair, asks for a passphrase and writes an encrypted key bundle to disk in the XDG compliant config directory of Holochain which is dependent on the OS platform (/home/alice/.config/holochain or C:\Users\Alice\AppData\Roaming\holochain\holochain or /Users/Alice/Library/Preferences/com.holochain.holochain)"
```

Reply:
the directory you mention is the XDG compliant one, but there is the `keys` subdir where it actually ends up. 
When you actually generate the keys, it shows you the exact location of it, so I thought it should be fine to leave it off/out in the first place, from the help text

Reply:
This way you see what it would be on other platforms..
Here's with /keys
```suggestion
        about = "Creates a new agent key pair, asks for a passphrase and writes an encrypted key bundle to disk in the XDG compliant config directory of Holochain, which is dependent on the OS platform (/home/alice/.config/holochain/keys or C:\Users\Alice\AppData\Roaming\holochain\holochain\keys or /Users/Alice/Library/Preferences/com.holochain.holochain/keys)"
```

Reply:
Ok cool. Good with that then